resources/migrations/20200921124315-create-invites.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS invites(
+       id bigserial primary key,
+       organization_id bigint not null references organizations(id),
+       email text not null
+);
+--;;
+CREATE UNIQUE INDEX IF NOT EXISTS invites_organization_id_email_idx
```

Comment:
Do we need to explicitly drop this index in down-migration?

Reply:
No, dropping the table removes the index as well.